src/components/App/App.js

```diff
@@ -38,11 +38,14 @@ class App extends React.Component {
     this.setState({ currentListing: listingId });
   };
 
+  // should we change to only store the id in favorites?? Otherwise we might be have double data in our state...if so we could have a function then that filters through the ids (findFavoritesToDisplay) and then set that to the props for favorite?
```

Comment:
Yes, I was thinking about this when I was writing the tests for Favorites. Let's work on refactoring this when we do the conditional rendering for the Favorites button.